<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSizeSelect</class>
 <widget class="QDialog" name="QSizeSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radio1024">
         <property name="text">
          <string>1024 x 768</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radio768">
         <property name="text">
          <string>768 x 1024</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radio480">
         <property name="text">
          <string>480 x 320</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radio320">
         <property name="text">
          <string>320 x 480</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="custom">
         <property name="text">
          <string>custom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="width">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="height">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ok">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
